bin/reindex

```diff
@@ -29,11 +30,31 @@ echo "Importing all API content"
 echo "Waiting for empty queues"
 ./bin/wait-for-empty-queues
 
+echo "Check for empty index"
+index_check=$(./bin/console index:total:write)
+
+if [ "$index_check" -eq "0" ] || true; then
```

Comment:
Do you think it's better to remove it ?

Reply:
Ok, then I think it would be better to keep only the `"$index_check" -eq "0"` check and that should be enough for now until we implement [#7982](https://github.com/elifesciences/issues/issues/7982)